Cargo.toml

```diff
@@ -35,8 +35,8 @@ iron = "^0.5"
 urlencoded = "^0.5"
 router = "^0.5"
 error-type = "^0.1"
-serde_json = "^0.8"
-spaceapi = "^0.4.2"
+spaceapi = "^0.5.0"
+serde_json = "^1.0"
```

Comment:
Why did you change the order of the crates :smile:? Also maybe we should re-export the needed serde_json stuff from spaceapi? Otherwise we need to match the version manually here.